codebuild_specs/mock_e2e_tests.yml

```diff
@@ -7,4 +7,5 @@ env:
 phases:
   build:
     commands:
+      - source shared-scripts.sh && _waitForJobs build_linux
       - source ./shared-scripts.sh && _mockE2ETests
```

Comment:
we only need to source shared-scripts once right?

Reply:
Unfortunately we've found that codebuild does not behave this wave; Each command has its own source/context with codebuild within the "commands" section